scripts/tasks/webpack.js

```diff
@@ -14,8 +14,9 @@ exports.webpackDevServer = async function() {
   if (fs.existsSync(configPath)) {
     const webpackDevServerPath = require.resolve('webpack-dev-server/bin/webpack-dev-server.js');
     const execSync = require('../exec-sync');
-
-    execSync(`node ${webpackDevServerPath} --config ${configPath} --port ${port} --open`);
+    const cmd = `node ${webpackDevServerPath} --config ${configPath} --port ${port} --open ${argv().cached ? '--env.cached=true' : ''}`;
+    console.log('RUNNING: ', cmd);
```

Comment:
Intentional?

Reply:
Not that I like this, but accidentally left this. Will convert to logger.info when the first iteration run is done to compare the yarn install speeds after build cache has been established.

Reply:
yarn build cache was slower!!